client/src/components/UserProfile.tsx

```diff
@@ -14,22 +16,25 @@ import Tab from '@mui/material/Tab';
 import TabPanel from '@mui/lab/TabPanel';
 import Skeleton from '@mui/material/Skeleton';
 
-
-const Profile = ():React.ReactElement => {
-
+const Profile = (): React.ReactElement => {
   const { id } = useParams();
-  const [profileData, setProfileData]: [UserProfile | null, Function] = useState(null);
-  const profileDataREF = useRef(profileData)
+  const [profileData, setProfileData]: [UserProfile | null, Function] =
+    useState(null);
+  const profileDataREF = useRef(profileData);
+  const [followerCount, setFollowerCount] = useState(0);
+  const [followingCount, setFollowingCount] = useState(0);

```

Comment:
Since these are calculated values, we can remove them from state values later. I think it would be as simple as including followers in the prisma fetch and the followers and following count would be the length.